apps/mobile/src/screens/signin/compose/signin-form.tsx

```diff
@@ -1,14 +1,18 @@
+import { useNavigation } from '@react-navigation/native';
 import { Button, Heading, Input, KeyboardAvoidingView, Text, View } from 'native-base';
 
 import { FormRow } from './form-row';
 import { useAuth } from '../../../hooks/use-auth-provider';
 import useLocale from '../../../hooks/use-locale';
+import Routes from '../../../routes';
 import theme from '../../../theme';
 
 export const SignInForm = () => {
```

Comment:
1. When this component renders, the password field is the first one that gets focused. Please, make sure the first field (name) is the focused one.
2. When I focus on the field, it shows me the error state (red outline) right away.